docs/Reference.md

```diff
@@ -887,9 +887,11 @@ A floating-point number that determines how quickly the scroll view decelerates
 - normal: 0.998
 - fast: 0.99 (the default for iOS web view)
 
-| Type   | Required | Platform |
-| ------ | -------- | -------- |
-| number | No       | iOS      |
+On Android you need to use only numeric values.
```

Comment:
```suggestion
> [!WARNING]
> On Android, string literal values such as `normal` and `fast` will not work. Use numeric values instead.
```

Reply:
GitHub's markdown warnings are a little more prominent 😀

Reply:
I just try to keep the same style